<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personwindow</class>
 <widget class="QWidget" name="personwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>name</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="personLocationLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="goingButton">
     <property name="text">
      <string>Going</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="nearbyCheckBox">
     <property name="text">
      <string>Show only nearby events (â‰¤ 5 units)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QListWidget" name="eventsList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Event description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Event name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descriptionLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="longLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="latLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dateLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add event</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
